_pages/SDK/v7.1/02-netsdk.md

```diff
@@ -0,0 +1,65 @@
+---
+area: sdk
+title: .NET SDK
+level: 0
+version: 7.1
+permalink: "/sdk/v7.1/"
```

Comment:
Why is this version 7.1 and not 7.0? Prior versions where 5.0 and 6.0.

Reply:
Breaking changes of NuGet v7.0 had no impact on the SDK documentation, so until now it was not updated. Since this new Feature comes with v7.1, I added this exact version so that this PR only adds relevant changes. I will add v7.0 in a separate PR.